src/highlight/src/extension.ts

```diff
@@ -500,7 +502,7 @@ function decorateActiveGuides() {
 }
 
 export function activate(context: vscode.ExtensionContext) {
-  activeEditor = getActiveTextEditor();
```

Comment:
Wow, I'm looking back through the code changes for this line and I _definitely_ picked the wrong function. This fixes it but this was definitely my mistake 😬

Reply:
It was bound to happen somewhere in all those changes. 😀 (Which were reviewed and approved by , btw.)

Not a huge harm done here, even. But great that the fix was so obvious and tiny.